spec/lib/rev/get_attachment_content_spec.rb

```diff
@@ -62,12 +62,11 @@
     action.must_raise Rev::NotFoundError
   end
 
-  # requesting conversion from pdf to :docx from order TC0263917003
```

Comment:
This PDF-to-docx test was confusing, I initially thought there was a valid test case we were accidentally removing, but it looks like it's just mocked behavior.